zkbind/src/org/zkoss/bind/impl/BinderImpl.java

```diff
@@ -293,6 +294,7 @@ private void init() {
 				_log.error("Error when initial phase listener:" + clz, e);
 			}
 		}
+		Executions.getCurrent().getDesktop().getWebApp().getConfiguration().registerCallBack("destroy", new DestroyCallback());
```

Comment:
Beware of NPE

Reply:
I put it in the BinderImpl, static{....} , to be sure that registerCallBack only register for once.
The stack is 
	  at org.zkoss.bind.impl.BinderImpl.<clinit>(BinderImpl.java:199)
	  at java.lang.Class.forName0(Class.java:-1)
	  at java.lang.Class.forName(Class.java:348)
	  at org.zkoss.lang.Classes.forNameByThread(Classes.java:274)
	  at org.zkoss.lang.Classes.newInstanceByThread(Classes.java:238)
	  at org.zkoss.bind.BindComposer.newAnnotateBinder(BindComposer.java:429)
	  at org.zkoss.bind.BindComposer.initBinder(BindComposer.java:414)
	  at org.zkoss.bind.BindComposer.doBeforeComposeChildren(BindComposer.java:177)
	  at org.zkoss.zk.ui.impl.UiEngineImpl.doBeforeComposeChildren(UiEngineImpl.java:991)
	  at org.zkoss.zk.ui.impl.UiEngineImpl.execCreateChild0(UiEngineImpl.java:931)
	  at org.zkoss.zk.ui.impl.UiEngineImpl.execCreateChild(UiEngineImpl.java:889)
	  at org.zkoss.zk.ui.impl.UiEngineImpl.execCreate0(UiEngineImpl.java:776)
	  at org.zkoss.zk.ui.impl.UiEngineImpl.execCreate(UiEngineImpl.java:740)
is after starting server, it no need to check the NPE.